secant/Dependencies/Deeplink/Deeplink.swift

```diff
@@ -10,31 +10,32 @@ import URLRouting
 import ComposableArchitecture
 import ZcashLightClientKit
 
-struct DeeplinkHandler {
-    enum Deeplink {
+struct Deeplink {
+    enum DeeplinkRoute: Equatable {
```

Comment:
This could be called only Route. `Deeplink.DeeplinkRoute` feel redundant. `Deeplink.Route` feels right.

Reply:
fixed